google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -310,27 +612,76 @@ class Page:
-            for line in self.documentai_page.lines
-        ]
-        self.paragraphs = [
-            Paragraph(documentai_paragraph=paragraph, document_text=document_text)
-            for paragraph in self.documentai_page.paragraphs
-        ]
-        self.blocks = [
-            Block(documentai_block=block, document_text=document_text)
-            for block in self.documentai_page.blocks
-        ]
-        self.tables = [
-            Table(documentai_table=table, document_text=document_text)
-            for table in self.documentai_page.tables
+            FormField(documentai_object=form_field, document_text=self.document_text)
+            for form_field in self.documentai_object.form_fields
         ]
+
+        self.tokens = _get_tokens(tokens=self.documentai_object.tokens, page=self)
+        self.lines = _get_lines(lines=self.documentai_object.lines, page=self)
+        self.paragraphs = _get_paragraphs(
+            paragraphs=self.documentai_object.paragraphs, page=self
+        )
+        self.blocks = _get_blocks(
+            blocks=self.documentai_object.blocks,
+            page=self,
+        )
+        self.tables = tables
+
+    def to_hocr(self, title):
+        r"""Exports a string hOCR version of the documentai.Document.Page.
+
+        Args:
+            title (str):
+                Required. The title for ocr_page.
+
+        Returns:
+            str:
+                A string hOCR version of the documentai.Document.Page.
+        """
+        f = ""
+        pidx = self.documentai_object.page_number
+        f += f"<div class='ocr_page' lang='unknown' title='image \"{title}\";{self.hocr_bounding_box}'>"
+        for bidx, block in enumerate(self.blocks):
+            f += f"<span class='ocr_carea' id='block_{pidx}_{bidx}' title='{block.hocr_bounding_box}'>"
+            for paridx, paragraph in enumerate(block.paragraphs):
+                f += f"<span class='ocr_par' id='par_{pidx}_{bidx}_{paridx}' title='{paragraph.hocr_bounding_box}'>"
+                for lidx, line in enumerate(paragraph.lines):
+                    line_text = html.escape(line.text)
+                    f += f"<span class='ocr_line' id='line_{pidx}_{bidx}_{paridx}_{lidx}' title='{line.hocr_bounding_box}'>{line_text}</span>"
+                    for tidx, token in enumerate(line.tokens):
+                        word_text = html.escape(token.text)
+                        f += f"<span class='ocrx_word' id='word_{pidx}_{bidx}_{paridx}_{lidx}_{tidx}' title='{token.hocr_bounding_box}'>{word_text}</span>"
+                f += "</span>"
+            f += "</span>"
+        f += "</div>"
+        return f
+
+    @property
+    def hocr_bounding_box(self):
+        if self._hocr_bounding_box is None:
+            self._hocr_bounding_box = _get_hocr_bounding_box(
+                element_with_layout=self.documentai_object,
+                dimension=self.documentai_object.dimension,
+            )
+        return self._hocr_bounding_box
```

Comment:
The hOCR needs to have all the information of the page and unlike all the other classes the hOCR exporter will not have any additions or features since it's only purpose is for exporting.